ElmersAPIServiceTests/NumToTextConverterTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using ElmersAPIService;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ElmersAPIService.Tests
+{
+    [TestClass()]
+    public class NumToTextConverterTests
+    {
+        [TestMethod()]
+        public void ConvertNumberToWords_ValidInput()
```

Comment:
DataRow and pass it the test cases as per the assessment

Reply:
Added DataRow Attributes for test cases defined in the assessment